src/txn/txn_rollback_to_stable.c

```diff
@@ -91,7 +93,23 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
         if (upd->txnid == WT_TXN_ABORTED)
             continue;
 
-        if (rollback_timestamp < upd->durable_ts || upd->prepare_state == WT_PREPARE_INPROGRESS) {
+        /*
+         * The unstable update according to the following needs to be aborted.
+         * 1. An invisible update according to the checkpoint snapshot during recovery.
```

Comment:
I am not sure what this line means. Maybe this (if it is accurate):
1. An update is invisible based on the checkpoint snapshot during recovery.

Reply:
Updated as per the suggestion.